src/Entity/EntityActionLogEntry.php

```diff
@@ -33,7 +33,7 @@ public function __construct(
         #[ORM\Column(name: 'context', type: Types::JSON, nullable: true)]
         private ?array $context = null,
     ) {
-        $this->createdAt = new \DateTimeImmutable('now', new \DateTimeZone('UTC'));
+        $this->createdAt = new \DateTime('now', new \DateTimeZone('UTC'));
```

Comment:
Whats the purpose of this change?

Reply:
https://github.com/itk-dev/dokk1gh/blob/6a117357a70081f7b071b2501ebb581bd20d573a/src/Entity/EntityActionLogEntry.php#L18
Either doctrine has become stricter or this has never worked …